src/ZkFold/Base/Data/Vector.hs

```diff
@@ -133,6 +134,11 @@ unsafeConcat = Vector . concatMap fromVector
 chunks :: forall m n a . KnownNat n => Vector (m * n) a -> Vector m (Vector n a)
 chunks (Vector lists) = Vector (Vector <$> chunksOf (fromIntegral $ value @n) lists)
 
+instance IsList (Vector size a) where
+    type Item (Vector size a) = a
+    fromList = unsafeToVector
```

Comment:
Do we actually want to call an "unsafe" function implicitly? Maybe use the safe alternative to cast from lists

Reply:
Removed